test/index.test.js

```diff
@@ -82,22 +82,25 @@ const specs = [
     input: 51,
     expected: 'st',
   },
-  {
-    description: 'Negative 51st',
-    input: -51,
-    expected: 'st',
-  },
 ];
 
 describe('thize', () => {
   specs.forEach(({ description, input, expected }) => {
-    it(description, () => assert.equal(thize(input), expected));
-    it(`Negative ${description}`, () => assert.equal(thize(-input), expected));
+    it(description, () => assert.equal(thize(input), input + expected));
```

Comment:
Now the `expected` is not really the "expected" result. Would it make more sense if we rename `expected` to `suffix`/`expectedSuffix`?

Reply:
you mean in the `specs` array?

Reply:
Ya, what do you think?